app/controllers/cities_controller.rb

```diff
@@ -5,7 +5,7 @@ class CitiesController < ApplicationController
   before_action :set_keepers, only: [:create, :update]
 
   def index
-    @cities = policy_scope(City).with_published_spots.alphabetize
+    @cities = policy_scope(City).alphabetize
```

Comment:
This action isn't hit, so changing this actually doesn't do anything for this page.

Reply:
What you'd like is to add `if @city.spots.published.blank?` to the view ..